src/DynamoPackages/PackageLoader.cs

```diff
@@ -399,6 +440,78 @@ public Package ScanPackageDirectory(string directory)
             return null;
         }
 
+        private static void CheckPackageNodeLibraryCertificates(string directory, Package discoveredPkg)
+        {
+            var dllfiles = new System.IO.DirectoryInfo(discoveredPkg.BinaryDirectory).EnumerateFiles("*.dll");
+            if (discoveredPkg.Header.node_libraries.Count() == 0 && dllfiles.Count() != 0)
+            {
+                throw new LibraryLoadFailedException(directory,
+                    String.Format(
+                        "A package called {0} found at {1} includes dll files but none are defined in node libraries in the package manifest.  Ignoring it.",
+                        discoveredPkg.Name, discoveredPkg.RootDirectory));
+            }
+
+            foreach (var manifestFile in discoveredPkg.Header.node_libraries)
```

Comment:
same with the name here - unless I am misunderstanding - personally I would get this confused with the manifest file itself - like the header file or the extensionManfiest